src/pal/src/init/pal.cpp

```diff
@@ -537,8 +537,6 @@ Initialize(
     SEHCleanup();
 CLEANUP5:
     PROCCleanupInitialProcess();
-CLEANUP4:
```

Comment:
I guess that historically, they were consecutive and as the code evolved, some get removed. The way it is now is very ugly and it would definitely be great to clean that stuff eventually.